### DISCUSSION TITLE
v1.4.8

### DISCUSSION THREAD

**Discussion by sgkoishi** [sgkoishi/PackTracker#10] [General]
Update 20.8.0.86601
Add revealed (or not?) `602` United in Stormwind
Add `737` Golden United in Stormwind
Update `637` Warlock Card Pack

<hr /><em>This discussion was created from the release <a href='https://github.com/sgkoishi/PackTracker/releases/tag/v1.4.8'>v1.4.8</a>.</em>

**Comment by bgb** [comment]
Hi, when are you going to release next version?

**Reply by sgkoishi** [reply]
The build is automated, you can click the [Actions](https://github.com/sgkoishi/PackTracker/actions) and go to the first row, and there's an item in the Artifacts list. [Click to download](https://github.com/sgkoishi/PackTracker/suites/3173537318/artifacts/73117141). Those artifacts will be kept for 6 months starting the day they were built.
I didn't change the version number; releases will match game updates or large changes.
